classes/controllers/FrmEntriesController.php

```
@@ -949,9 +945,8 @@ public static function show_entry_shortcode( $atts ) {
 	 * @return void
 	 */
 	public static function entry_sidebar( $entry = false ) {
-		$data = array();
-		$id   = 0;
-
+		$data        = array();
+		$id          = 0;
```

PHPMD (2.15.0)

Avoid unused local variables such as '$id'. (undefined)

(UnusedLocalVariable)